specification.md

```diff
@@ -3796,29 +3776,59 @@ You can combine retry and onError definitions to define powerful error handling
 
 ```json
 {
-"retry": [
+"retries": [
   {
-    "expression": "{{ $.errors[?(@.name == 'FunctionExecutionError')] }}",
-    "maxAttempts": 3,
-    "interval": "PT2M"
-  },
-  {
-    "expression": "{{ $.errors[?(@.name != 'FunctionExecutionError')] }}",
-    "maxAttempts": 2,
-    "interval": "PT1M"
-  }
-],
-"onError": [
-  {
-    "expression": "{{ $.errors[?(@.name == 'FunctionExecutionError')] }}'",
+    "name": "Service Call Timeout Retry Strategy",
+    "delay": "PT1M",
+    "maxAttempts": 4,
     "transition": {
-      "nextState": "afterFunctionErrorState"
+      "nextState": "handleServiceCallTimeout"
     }
-  },
+  }
+]
+}
+```
+
+</td>
+<td valign="top">
+
+```yaml
+retries:
+- name: Service Call Timeout Retry Strategy
+  delay: PT1M
+  maxAttempts: 4
+  transition:
```

Comment:
Also typo here to list `transition` parameter here, right?

Reply:
yeah i forgot to remove after cut/paste sry